cleanrepo/CleanRepo/Program.cs

```diff
@@ -1570,13 +1678,13 @@ private static List<FileInfo> GetYAMLFiles(string directoryPath)
     /// </summary>
     private static Dictionary<string, List<string>> GetMediaFiles(string mediaDirectory, bool searchRecursively = true)
     {
-        DirectoryInfo dir = new(mediaDirectory);
+        DirectoryInfo dir = new DirectoryInfo(mediaDirectory);
 
         SearchOption searchOption = searchRecursively ? SearchOption.AllDirectories : SearchOption.TopDirectoryOnly;
 
         Dictionary<string, List<string>> mediaFiles = new(StringComparer.InvariantCultureIgnoreCase);
 
-        string[] fileExtensions = ["*.png", "*.jpg", "*.gif", "*.svg"];
+        string[] fileExtensions = { "*.png", "*.jpg", "*.gif", "*.svg" }; // Correctly initialize the array
```

Comment:
The square brackets is "collection expressions", new for .NET 8/C# 12. Can you revert and check what .NET SDK version you have installed?

Reply:
`8.0.302 [C:\Program Files\dotnet\sdk]` - what version should I have installed?

Reply:
@BillWagner Any idea why Dina would see compile errors for collection expressions? The project targets .NET 8.

Reply:
@gewarren @BillWagner I'm probably doing something wrong - VS + .NET - what version of sdk should I have installed. Maybe I can just uninstall the wrong one and install the right one.

Reply:
@diberry If you're developing using Visual Studio, you need to be using 17.8 or later. (The side-by-side installation means that inside Visual Studio, it uses the associated SDK from that version of VS)